app_flutter/test/utils/wrapper.dart

```diff
@@ -2,9 +2,11 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:flutter/material.dart';
```

Comment:
Thank you for bringing order to this file! :)

Reply:
well now that i know what convention we're following... :-)